### ISSUE TITLE
Paper mentions alpha=10 and beta=1

### ISSUE THREAD

**Issue by dpt-xyz** [jxbbb/TOD3Cap#21]
Hi,  great work!
I have a query regarding the weighted combination used in overall loss computation.

The loss computation section of paper mentions that overall loss for dense captioning is calculated as the weighted combination of object detection loss and captioning loss where the weightage of object detection loss is set to 10 and the weightage of captioning loss is set to 1 (as shown in thr screenshot).
However,  I couldn't figure where these values are set in the code. 
To the best of my understanding,  the weightage for object detection loss is set to 0.3 and the weightage for captioning loss is set to 2 in the code.
Please correct me if I'm wrong. 

![Image](https://github.com/user-attachments/assets/49ac6cb8-9a3a-4fcb-9e4b-cc66d769d486)

**Comment by ayankashyap** [comment]
I was confused about this as well, hope the authors clarify

**Comment by ucaszyp** [comment]
> Hi, great work! I have a query regarding the weighted combination used in overall loss computation.
> 
> The loss computation section of paper mentions that overall loss for dense captioning is calculated as the weighted combination of object detection loss and captioning loss where the weightage of object detection loss is set to 10 and the weightage of captioning loss is set to 1 (as shown in thr screenshot). However, I couldn't figure where these values are set in the code. To the best of my understanding, the weightage for object detection loss is set to 0.3 and the weightage for captioning loss is set to 2 in the code. Please correct me if I'm wrong.
> 
> ![Image](https://github.com/user-attachments/assets/49ac6cb8-9a3a-4fcb-9e4b-cc66d769d486)

Thanks for your interest!
After submitting the paper, we have optimized the code, including the model structure, training strategy, and some hyperparameters such as loss weight. Therefore, some of the current hyperparameters are mismatched with the paper. The results of reproducing the code in the current repository should be much better than those in the paper.
